src/components/movie-card/movie-card.jsx

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import {Button, Card} from 'react-bootstrap';
+
+import { Link } from "react-router-dom";
 
 export class MovieCard extends React.Component {
   render() {
-    const { movie, onMovieClick } = this.props;
-    return <div className="movie-card" onClick={()=>{onMovieClick(movie)}}>{movie.Title}</div>;
+    const { movie } = this.props;
+    return (
+     
+      <Card border="dark" className='mb-3'>
+        <Card.Img variant="top" src={movie.imageURL} />
+        <Card.Body className="d-flex flex-column justify-content-between align-items-start">
+          <div>
+            <Card.Title>{movie.Title}</Card.Title>
+            <Card.Text>{movie.Description}</Card.Text>
+          </div>
+          <Link to={`/movies/${movie._id}`}>
+            <Button variant="link" className='mt-2'>Open</Button>
```

Comment:
Hi Shuang, you could add the add fav movie button here as well, so users can add their fav movies directly from the list of the movie. 
Thank you!